packages/babel-node/src/babel-node.ts

```diff
@@ -96,7 +97,7 @@ getV8Flags(async function (err, v8Flags) {
         if (signal) {
           process.kill(process.pid, signal);
         } else {
-          process.exitCode = code;
+          process.exitCode = code || undefined;
```

Comment:
```suggestion
          process.exitCode = code ?? undefined;
```
We may not want to cast `0` to `undefined`.